Core/Core/Credentials/StreamWrapper.cs

```diff
@@ -207,13 +207,13 @@ private void StreamWrapperFromUrl(string streamUrl)
       switch (uri.Segments.Length)
       {
         case 3: // ie http://speckle.server/streams/8fecc9aa6d
-          if (uri.Segments[1].ToLowerInvariant() == "streams/")
+          if (uri.Segments[1].ToLowerInvariant() != "streams/")
           {
-            StreamId = uri.Segments[2].Replace("/", "");
+            throw new SpeckleException($"Cannot parse {uri} into a stream wrapper class.");
           }
           else
           {
-            throw new SpeckleException($"Cannot parse {uri} into a stream wrapper class.");
+            StreamId = uri.Segments[2].Replace("/", "");
           }
```

Comment:
I'm amazed this was not flagged as redundant.

Reply:
Which? the `else`?

Not going to lie, I don't remember making this change. Perhaps it was a simple rider refactor preferring exceptional paths first.
Happy to revert if you'd prefer